test/src/components/traces/traces.spec.jsx

```diff
@@ -305,16 +331,26 @@ function createStubStore(results, promise, searchQuery = {}) {
     return store;
 }
 
-function createStubDetailsStore(spans, promise) {
+function createStubDetailsStore(spans, promise, relatedTracesPromise = fulfilledPromise, relatedResults = []) {
     const store = new TraceDetailsStore();
     sinon.stub(store, 'fetchTraceDetails', () => {
-        store.spans = spans;
+        store.spans = observable.array(spans);
         store.promiseState = promise;
     });
+    sinon.stub(store, 'fetchRelatedTraces', (query) => {
+        store.relatedTraces = observable.array(relatedResults);
+        store.relatedTracesPromiseState = relatedTracesPromise;
+        store.searchQuery = query;
+    });
 
     return store;
 }
 
+before((done) => {
+    ReactGA.initialize('foo', { testMode: true });
+    done();
+});
+
```

Comment:
This script initializes ReactGA in test mode, which resolves the warning messages in tests about react-ga not be initialized.